### ISSUE TITLE
pull request template

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/nfdi4plants.knowledgebase#2]
Create a default pull request template that links to 
- contribution guide (bigger changes)
- landing page (small changes)

**Comment by Freymaurer** [comment]
GitHub also features a [CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).

If you could organize the CONTRIBUTING.md together with our contribution guide and the landing page, that would be great.

And then accordingly create the PR template.

**Comment by Brilator** [comment]
Didn't I send you that link yesterday and asked the same thing? :D

**Comment by Brilator** [comment]
Lost in Teams, but good idea to have the repo-relevant discussion here. 👍

**Comment by Brilator** [comment]
So, just to be clear. 
- [ ] We formulate a contributing.md (in root or in /docs?)
- [ ] where we link 1. the home / index / landing page and 2. the /docs/README
- [ ] + "Steps for creating good issues or pull requests."

**Comment by Freymaurer** [comment]
Maybe contributing.md is a better fit for all contributions so other docs pages (web-component-docs) can just reference it for theirs.

**Comment by Brilator** [comment]
Yes, actually this would potentially also be relevant for (most) other repositories, including tools, etc. 
So maybe design it even one level higher, incl. code of conduct, etc.
